tests/test-quickstart/pom.xml

```diff
@@ -102,7 +102,14 @@
       <groupId>org.eclipse.jetty.toolchain</groupId>
       <artifactId>jetty-test-helper</artifactId>
     </dependency>
-
+    <dependency>
+      <groupId>org.slf4j</groupId>
+      <artifactId>slf4j-api</artifactId>
+    </dependency>
+    <dependency>
+      <groupId>org.eclipse.jetty</groupId>
+      <artifactId>jetty-slf4j-impl</artifactId>
+    </dependency>
```

Comment:
Both should have `scope=test`.

Reply:
`test-quickstart` has no test scoped dependencies currently, despite it consisting of 100% `src/test/java`.
guess I'll make all of it `<scope>test</scope>` then.